whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/domain/NServerValidator.java

```diff
@@ -44,6 +48,8 @@ public void validate(final PreparedUpdate update, final UpdateContext updateCont
                 case INADDR:
                 case IP6:
                 {
+                    validateNserverCorrectPrefixes(update.getUpdatedObject(), domain.getReverseIp().getPrefixLength(),
+                            domain.getType().equals(INADDR));
                     final boolean endsWithDomain = domain.endsWithDomain(nServer.getHostname());
```

Comment:
can we have this check after hostname validation is performed as ipInterval can be null and it should fail before that

Reply:
in Domain.endsWithDomain if isDasNotation is false, we still could have the problem that the reverseIp is null. 
I will add it before that and add an if